tests/postgres/trawselect.nim

```diff
@@ -0,0 +1,108 @@
+import std/[unittest, with, os, sugar, options, logging, strutils]
+
+import norm/[model, postgres]
+
+import ../models
+
+
+const
+  dbHost = "postgres"
+  dbUser = "postgres"
+  dbPassword = "postgres"
+  dbDatabase = "postgres"
+
+proc resetDb =
+  let dbConn = open(dbHost, dbUser, dbPassword, "template1")
+  dbConn.exec(sql "DROP DATABASE IF EXISTS $#" % dbDatabase)
+  dbConn.exec(sql "CREATE DATABASE $#" % dbDatabase)
+  close dbConn
+
+
+addHandler(newConsoleLogger(levelThreshold = lvlDebug))
+
+suite "Testing rawSelect proc":
+  setup:
+    resetDb()
+    let dbConn = open(dbHost, dbUser, dbPassword, dbDatabase)
+
+    var
+      alice = newPerson("Alice", none Pet)
+      bob = newPerson("Bob", none Pet)
+      jeff = newPerson("Jeff", none Pet)
+
+      someDoctor = newDoctor("Vet1")
+
+      visit1 = newDoctorVisit(alice, someDoctor)
+      visit2 = newDoctorVisit(bob, someDoctor)
+
+      boneToy = newToy()
+      ballToy = newToy()
+
+      spot = newPlayfulPet("spot", boneToy, ballToy)
+
+    dbConn.createTables(newPerson())
+    dbConn.createTables(newDoctor())
+    dbConn.createTables(newDoctorVisit())
+    dbConn.createTables(newToy())
+    dbConn.createTables(newPlayfulPet())
+    discard @[alice, bob, jeff].dup:
+      dbConn.insert
+
+    discard @[someDoctor].dup:
+      dbConn.insert
+
+    discard @[visit1, visit2].dup:
+      dbConn.insert
+
+    discard @[boneToy, ballToy].dup:
+      dbConn.insert
+
+    discard @[spot].dup:
+      dbConn.insert
+
+
+  teardown:
+    close dbConn
+    resetDb()
+
+
+  test "rawSelect single entry":
+    type MyDoc = ref object
+      name: string
+
+    let sql = """SELECT name FROM "Doctor" WHERE id = $1"""
+    var mydoc = MyDoc()
+    dbConn.rawSelect(sql, mydoc, someDoctor.id)
+
+    check mydoc.name == "Vet1"
+
+  test "rawSelect multiple entries":
+    type MyPerson = ref object
+      name: string
+
+    let sql = """SELECT name FROM "Person" ORDER BY id"""
```

Comment:
I've added some basic tests now and updated the changelog.

Sidenote that I noticed in that regard @moigagoo : 
When using `rawSelect` with postgres, the query would not recognize the table name without the `"`.
As in, this would not work: `SELECT name FROM Person ORDER BY id` 
But this would: `SELECT name FROM "Person" ORDER BY id`
This is only this way in **postgres**

Looking at the table names via
```sql
SELECT table_name FROM information_schema.tables WHERE table_schema='public' AND table_type='BASE TABLE';
```
Revealed nothing suspicious that could hint as to why the quotation marks are mandatory:
```txt
+@[@[Toy], @[Pet], @[Person], @[DoctorVisit], @[Doctor], @[PlayfulPet]]
```

Any ideas?

Reply:
I think it might have to do with capitalized table names: https://stackoverflow.com/questions/43111996/why-postgresql-does-not-like-uppercase-table-names

Quotes should be used everywhere for consistency in both Postgresl and SQLite.

Reply:
If that is how it is intended to be, then I should document this behaviour in the doc comment of the proc.
The most recent commit just did that.
Since SQLite does not display this requirement behaviour I did not add that same info to the sqlite proc.